tests/util/checker/parser/testdata/flaky_rule_test.go

```diff
@@ -0,0 +1,42 @@
+package testdata
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/annotation"
+)
+
+func SetCount()       {}
+func Count(n int) int { return n }
+
+// nolint: testlinter
+func TestIsMarkedFlaky1(t *testing.T) {
```

Comment:
Add a non flaky function and make sure it is not matched?

Reply:
Added. Thanks!